global/sass-config/_colors.sass

```diff
@@ -1,15 +1,17 @@
 // BY NAME
 // Define all your color values by color names here
-$black: #000
-$white: #fff
-$nav-background: #0f0f0f
+$brand-color: #325B31
```

Comment:
So these should go in the color-by-purpose section and reference color name variables.
So $color-dark-green: #325B31 in this section
And $brand-color: $color-dark-green in the by purpose section.